Libraries/src/Amazon.Lambda.SQSEvents/SQSBatchResponse.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.Generic;
+
+namespace Amazon.Lambda.SQSEvents
+{
+    /// <summary>
+    /// Simle Queue Service response
```

Comment:
```suggestion
    /// This class can be used as the return type for Lambda functions that have partially succeeded by supplying a list of message IDs that have failed to process.
    /// https://docs.aws.amazon.com/lambda/latest/dg/with-sqs.html#services-sqs-batchfailurereporting
```

Reply:
I'm not great with wording, but something like this

Reply:
I think that works, I see my typo above.  I had kept with the general style on the SQSEvent class, but I always like more comments myself.